qemu/tests/virtio_serial_large_file_transfer.py

```diff
@@ -0,0 +1,66 @@
+import os
+import time
+import signal
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import data_dir
+
+from qemu.tests.virtio_serial_file_transfer import generate_data_file
+from qemu.tests.virtio_serial_file_transfer import get_command_options
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test long time virtio serial guest file transfer.
+
+    Steps:
+    1) Boot up a VM with virtio serial device.
+    2) Create a large file in guest or host(sender).
+    3) In 'repeat_time',  repeatedly transfer the file between guest and host
+    4) Kill the data transfer process
+    5) Check guest running well, no crash
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    sender = params['file_sender']
+    file_size = int(params.get("filesize", 100))
+    repeat_time = int(params.get("repeat_time", 600))
```

Comment:
Seems there is no parameter 'repeat_time' in Dic params.

Reply:
@nanliu-r Yes, so it utilizes the default value `600`, just leave an interface to customize the parameter, users can define it when needed.